numbers/number.py

```diff
@@ -1,14 +1,19 @@
-def add(a, b):
-    pass
-
-
-def is_even(n):
-    pass
-
-
-def is_odd(n):
-    pass
-
-
-def sum_all(arr):
-    pass
+def add(a, b):
+    return(a+b)

```

Comment:
no hacen falta los paréntesis acá 
```suggestion
    return a + b

```